package.json

```diff
@@ -14,17 +14,22 @@
     "generate-inference-client": "openapi-ts --input http://localhost:8000/openapi.json --output ./src/apis/inference --client @hey-api/client-axios"
   },
   "dependencies": {
+    "@coral-xyz/anchor": "^0.31.1",
     "@hey-api/client-axios": "^0.7.0",
-    "@radix-ui/react-dialog": "^1.1.14",
-    "@radix-ui/react-dropdown-menu": "^2.1.15",
-    "@radix-ui/react-popover": "^1.1.14",
-    "@radix-ui/react-separator": "^1.1.7",
-    "@radix-ui/react-slot": "^1.2.3",
-    "@radix-ui/react-switch": "^1.2.5",
-    "@radix-ui/react-tooltip": "^1.2.7",
-    "@tailwindcss/vite": "^4.1.11",
-    "@tanstack/react-query": "^5.83.0",
-    "@tanstack/react-router": "^1.128.4",
+    "@radix-ui/react-dialog": "^1.1.11",
+    "@radix-ui/react-dropdown-menu": "^2.1.12",
+    "@radix-ui/react-popover": "^1.1.11",
+    "@radix-ui/react-separator": "^1.1.4",
+    "@radix-ui/react-slot": "^1.2.0",
+    "@radix-ui/react-switch": "^1.2.2",
+    "@radix-ui/react-tooltip": "^1.2.4",
+    "@solana/spl-token": "^0.4.13",
+    "@solana/wallet-adapter-react": "^0.15.39",
+    "@solana/wallet-adapter-react-ui": "^0.9.39",
+    "@tailwindcss/vite": "^4.1.4",
+    "@tanstack/react-query": "^5.74.4",
+    "@tanstack/react-router": "^1.117.1",
+    "buffer": "^6.0.3",
```

Comment:
why do you need buffer ? If needed add https://www.npmjs.com/package/vite-plugin-node-polyfills (check the landing site)